From 63d18ce416d23580be88dfcaaad79916fba5e6bd Mon Sep 17 00:00:00 2001
From: Zhang Xiaodong <B39117@freescale.com>
Date: Wed, 20 Mar 2013 16:06:35 +0800
Subject: ENGR00254896 mx6: hdmidongle: Fix compile error.

After enable PICE in kernel config, building will meet
fatal error:linux/wakelock.h: No such file or directory

Signed-off-by: Zhang Xiaodong <B39117@freescale.com>
---
 arch/arm/mach-mx6/board-mx6q_hdmidongle.c |    7 -------
 1 file changed, 7 deletions(-)

diff --git a/arch/arm/mach-mx6/board-mx6q_hdmidongle.c b/arch/arm/mach-mx6/board-mx6q_hdmidongle.c
index 1a06774..ffd8d9a 100644
--- a/arch/arm/mach-mx6/board-mx6q_hdmidongle.c
+++ b/arch/arm/mach-mx6/board-mx6q_hdmidongle.c
@@ -68,10 +68,6 @@
 #include <asm/mach/arch.h>
 #include <asm/mach/time.h>
 
-#ifdef CONFIG_IMX_PCIE
-#include <linux/wakelock.h>
-#endif
-
 #ifdef CONFIG_MFD_WM831X
 #include <linux/mfd/wm831x/core.h>
 #include <linux/mfd/wm831x/pdata.h>
@@ -105,7 +101,6 @@
 #define HDMIDONGLE_PCIE_RST     IMX_GPIO_NR(3, 9)
 #define HDMIDONGLE_PCIE_WAKE	IMX_GPIO_NR(3, 22)
 #define HDMIDONGLE_PCIE_DIS		IMX_GPIO_NR(3, 10)
-static struct wake_lock pcie_wake_lock;
 #endif
 
 extern char *gp_reg_id;
@@ -735,8 +730,6 @@ static void __init mx6_hdmidongle_board_init(void)
 		/* Add PCIe RC interface support */
 #ifdef CONFIG_IMX_PCIE
 		imx6q_add_pcie(&mx6_hdmidongle_pcie_data);
-		wake_lock_init(&pcie_wake_lock, WAKE_LOCK_SUSPEND, "pcie_workaround");
-		wake_lock(&pcie_wake_lock);
 #endif
 	}
 	pm_power_off = mx6_snvs_poweroff;
-- 
1.7.9.5

